src/routes/(infoPages)/[topTierPage]/[...serviceGroupPage]/+page.svelte

```diff
@@ -13,6 +13,8 @@
   import Image from "$lib/components/Image";
   import Link from "$lib/components/Link";
   import { url as arrowIcon } from "$icons/arrow_forward";
+  import NewsEntrySnippet from "../../about/news/NewsEntrySnippet.svelte";
+  import Event from "../../about/events/Event.svelte";
```

Comment:
now that this is getting used across multiple pages, might make sense to move it under with `$lib/components` or another shared directory

Reply:
Yeah, that would definitely be ideal. I think at this point though a refactor would require a rather large lift, and it could spiral pretty quickly. I vote we keep it as-is for now to keep this PR simple, and we can tackle the refactor later (hopefully, if we have time).